### PR TITLE
[CDAP-14007] Adds UI support for bi-directional field level lineage

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14007
Build: https://builds.cask.co/browse/CDAP-UDUT91

TODO (in follow up PR):
- Update EntityTopPanel: https://issues.cask.co/browse/CDAP-14294

### CODE REVIEW

File: cdap-ui/app/cdap/components/DatasetDetailedView/index.js [hunk 1]
```diff
@@ -234,28 +233,17 @@ export default class DatasetDetailedView extends Component {
...
-                <DatasetDetaildViewTab
-                  params={this.props.match.params}
-                  pathname={this.props.location.pathname}
-                  entity={this.state.entityDetail}
-                />
-              );
-            }
-          } />
-        </Switch>
+        <Route path="/ns/:namespace/datasets/:datasetId/" render={
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DatasetDetailedView/index.js]: nit: props on newline?

File: cdap-ui/app/cdap/components/FieldLevelLineage/Lineage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  lineage: boolean;
+}
+
+interface ILineageViewProps {
+  fields: IField[];
+  datasetId: string;
+}
+
+const LineageView: React.SFC<ILineageViewProps> = ({fields, datasetId}) => {
+  if (fields.length === 0) {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FieldLevelLineage/Lineage/index.tsx]: Will this affect the search? If I search for a field will it remove the entire lineage tables?

**Review by ajainarayanan — DISMISSED** [review]
👍

**Comment by elfenheart** [comment]
Latest Build: https://builds.cask.co/browse/CDAP-UDUT91-6

**Review by ajainarayanan — APPROVED** [review]
👍

**Pull request merged**